src/components/RichTextPopup/RichTextPopup.js

```diff
@@ -40,27 +43,43 @@ const RichTextPopup = () => {
     };
 
     core.addEventListener('editorSelectionChanged', handleSelectionChange);
-    return () => core.removeEventListener('editorSelectionChanged', handleSelectionChange);
+    return () =>
+      core.removeEventListener('editorSelectionChanged', handleSelectionChange);
   }, []);
 
   useEffect(() => {
     const handleTextChange = () => {
       if (annotationRef.current?.isAutoSized() && popupRef.current) {
-        const position = getRichTextPopupPosition(annotationRef.current, popupRef);
+        const spaceAdjustment = symbolsVisible ? symbolsAreaHeight : 0;
+        const position = getRichTextPopupPosition(
+          annotationRef.current,
+          popupRef,
+          spaceAdjustment
+        );
         setCssPosition(position);
       }
 
       setFormat(getFormat(editorRef.current?.getSelection()));
     };
 
     core.addEventListener('editorTextChanged', handleTextChange);
-    return () => core.removeEventListener('editorTextChanged', handleTextChange);
+    return () =>
+      core.removeEventListener('editorTextChanged', handleTextChange);
   }, []);
 
   useEffect(() => {
     const handleEditorFocus = (editor, annotation) => {
-      if (annotation instanceof window.Annotations.FreeTextAnnotation && popupRef.current) {
-        const position = getRichTextPopupPosition(annotation, popupRef);
+      if (
+        annotation instanceof window.Annotations.FreeTextAnnotation &&
+        popupRef.current
+      ) {

```

Comment:
Same comment here. `spaceAdjustment` is always 0

Reply:
spaceAdjustment has been removed. I update `handleSymbolsClick` to adjust the popup position every time user show/hide the math symbols.  I feel the app is more smooth in this way.